src/auto-complete.jsx

```diff
@@ -152,6 +157,7 @@ const AutoComplete = React.createClass({
       open: false,
       searchText: '',
       menuCloseDelay: 100,
+      menuItemsLimit: 20,
```

Comment:
Initially I was thinking at 0 to keep the type of the property consistent but at the end it doesn't really matter for me